extensions/amp-story/1.0/amp-story-embedded-component.js

```diff
@@ -615,30 +634,33 @@ export class AmpStoryEmbeddedComponent {
 
           // Gap on the left of the element between full-screen size and
           // current size.
-          const leftGap = (storedStyle.width - targetRect.width) / 2;
+          const leftGap = (embedData.width - targetRect.width) / 2;
           // Distance from left of page to what will be the left of the
           // element in full-screen.
           const fullScreenLeft = targetRect.left - leftGap - pageRect.left;
-          const centeredLeft = pageRect.width / 2 - storedStyle.width / 2;
+          const centeredLeft = pageRect.width / 2 - embedData.width / 2;
           state.translateX = centeredLeft - fullScreenLeft;
 
           // Gap on the top of the element between full-screen size and
           // current size.
-          const topGap = (storedStyle.height - targetRect.height) / 2;
+          const topGap = (embedData.height - targetRect.height) / 2;
           // Distance from top of page to what will be the top of the element in
           // full-screen.
           const fullScreenTop = targetRect.top - topGap - pageRect.top;
-          const centeredTop = pageRect.height / 2 - storedStyle.height / 2;
+          const centeredTop = pageRect.height / 2 - embedData.height / 2;
           state.translateY = centeredTop - fullScreenTop;
         },
         /** mutate */
         () => {
           target.classList.toggle('i-amphtml-expanded-component', true);
 
-          const styleEl = embedStyleEls[embedId];
-          embedStyleEls[embedId].textContent = styleEl.textContent.split(
-              `scale(${storedStyle.scale})`).join(`scale(1)
-              translate3d(${state.translateX}px, ${state.translateY}px, 0)`);
+          const expandingData = Object.assign({}, embedData);
+          expandingData.translateX = state.translateX;
```

Comment:
nit: might be more readable to just include this as another object:

```javascript
  const expandingData = Object.assign({}, embedData, {
    translateX: state.translateX,
    translateY: state.translateY,
    scale: 1,
  });
```

(same thing below in `prepareForAnimation`)

Reply:
Done.